src/styles/transitions/_text-page-load.scss

```diff
@@ -0,0 +1,22 @@
+.text-page-load {
+  max-width: 590px;
+  display: block;
+  margin: auto;
+}
+
+.text-page__title-load{
+  height: 32px;
+  margin: 0 0 0.5rem 0;
+  border-radius: 10px;
```

Comment:
I think it looks better without the border-radius. Try to make it look more like the image attached to the issue (https://github.com/thegazelle-ad/gazelle-server/issues/138)